byo-mgmt/registration/near-edge/base/argocd-application.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: application
+  namespace: openshift-gitops
+spec:
+  project: default
+  source:
+    path: acm/odh-edge/apps/app
+    repoURL: 'https://github.com/opendatahub-io/ai-edge.git'
+    targetRevision: main
```

Comment:
If trying this out locally, one might have more success changing these values to be the `grdryn` fork, and the `RHOAIENG-2787_byo-mgmt` branch from this PR.